contents/newsletter/small-teams.md

```diff
@@ -0,0 +1,96 @@
+---
+title: The secret to shipping really fast
+date: 2024-05-28
+author:
+  - james-temperton
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/v1711656184/posthog.com/contents/images/newsletter/remote-work/remote_hog.png
+featuredImageType: full
+tags:
+  - Culture
+crosspost:
+  - Founders
+  - Blog
+---
+Startups ship more stuff per person than big companies. At PostHog, we ship fast even for a startup. How? The magic of small teams. Basically, we’re growing as a company by creating a bunch of internal startups to keep us speedy, autonomous, and innovative. 
+
+This allows us to scale without falling into an organizational trap of having lots of big, separate functions that slow down communication, stifle innovation, and stop people from seeing the impact of their work.
+
+This is how small teams work, and why they’re important.
```

Comment:
```suggestion
Yes, there are tradeoffs and the small teams structure won't work for everyone. But, right now, it works for us. And maybe it could work for you and your company, too. 

This is how small teams work, and why they’re important.
```

Reply:
- I'd drop the 'And maybe it could for you... etc.' bit.
- It's comms and support, not community afaik.
- Maybe the web analytics example isn't a great one? Afaik, I think James and Tim came to conclusion teams of 1 are best avoided in future. Also, Robbie is technically part of the product analytics team as well, afaik.